new-client/src/plugins/collector/components/Page.js

```diff
@@ -568,13 +573,39 @@ class Page extends Component {
         onClick={() => {
           this.props.model.observer.publish("abort");
         }}
+      >
+        {okButtonText}
```

Comment:
Drop the `okButtonText` variable and just do a conditional render like this:
`this.props.options.collectAgain ? "Tyck till igen : "Stäng"`